### DISCUSSION TITLE
About contribution rules

### DISCUSSION THREAD

**Discussion by sjw-pc** [tinymce/tinymce#7905] [General]
How to contribute code
Do I need to create a new branch named ‘TINY-xxxx’？
If so, how to define the number,for example'TINY-8900'
Is there an automatic generation method

**Comment by TheSpyder** [comment]
For third party contributions you can use whatever branch name you like on a fork in your GitHub account. The `TINY-1234` codes we use for our branches are references to a private JIRA server. They are not related to issue numbers, although we do link the references internally.

**Reply by sjw-pc** [reply]
Thanks!
